tests/DependenciesTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * This file is part of the Fedora Autoloader package.
+ *
+ * (c) Shawn Iwinski <shawn@iwin.ski> and Remi Collet <remi@fedoraproject.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Fedora\Autoloader\Test;
+
+use Fedora\Autoloader\Dependencies;
+
+class DependenciesTest extends \PHPUnit_Framework_TestCase
+{
+    public function testRequiredExists()
+    {
+        $this->assertFalse(class_exists('Foo\\Bar'));
+        $this->assertFalse(class_exists('Foo\\Bar\\Baz'));
+
+        Dependencies::required(array(
+            __DIR__.'/fixtures/Foo/Bar.php',
+            __DIR__.'/fixtures/Foo/Bar/Baz.php',
+        ));
+
+        $this->assertTrue(class_exists('Foo\\Bar'));
+        $this->assertTrue(class_exists('Foo\\Bar\\Baz'));
+    }
+
+    public function testRequiredNotExists()
+    {
+        $this->markTestSkipped('Skipping until we can get this test to pass');
+        //Dependencies::required(array(__DIR__.'/fixtures/DoesNotExist.php'));
+    }
```

Comment:
Can this be tested by a @expectedException PHPUnit_Framework_Error ?
See: https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.errors